style.css

```diff
@@ -0,0 +1,184 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Poppins, sans-serif;
+  margin: 0;
+}
```

Comment:
- [ ] they is a horizontal scroll in the project mobile view, you can remove this by add `overflow: hidden' to `body`

Reply:
If I do it, the user will not be able to see the whole page when he wants to scroll down. Thanks for the feedback